lib/human.rb

```diff
@@ -0,0 +1,22 @@
+module TicTacToe
+  class Human < Player
+
+    def make_move(board)
+      @console_ui.move_messages(board.cells, @marker)
+      @console_ui.move_selection_message
+      move = @console_ui.user_input
+      move = valid_move_loop(move, board)
+      board.fill(move, @marker)
```

Comment:
I think this was confusing on how I had written it. `board.fill` is returning a `Board` object. I changed `Computer` to have the last line of the method match `Human` which hopefully makes it more clear. But let me know if you suggest doing it another way.